### ISSUE TITLE
Unable to reproduce Figure 6 on memorization effects

### ISSUE THREAD

**Issue by gordon-lim** [UCSC-REAL/cifar-10-100n#9]
I am not able to reproduce the trends in Figure 6. My ResNet50 only starts fitting to the noisy labels after a learning rate update. Could you share additional hyperparameters perhaps not discussed in the paper that I would need to reproduce your result? I also have the following questions regarding the split:
1. "we split the train data into two parts: images with clean labels (the annotation matches the clean label) and wrong labels (the rest)". Are there then actually 4 splits total? 2 splits for human noised and another 2 splits for synthetic noised?
2. If the top row figures are the split where the noised labels match the clean labels (i.e these splits have clean labels) why is there a difference between the trends for real vs. synthetic?

If possible, I think providing code that generated those plots is the best way to bring full clarity. Thank you.
